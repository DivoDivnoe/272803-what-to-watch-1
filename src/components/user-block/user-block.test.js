import React from 'react';
import {create} from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import UserBlock from './user-block.jsx';

const mock = {
  userData: {
    avatarUrl: ``,
  }
};

describe(`UserBlock component`, () => {
  it(`renders correctly`, () => {
    const {userData} = mock;

    const tree = create(
        <BrowserRouter>
          <UserBlock userData={userData} />
        </BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
