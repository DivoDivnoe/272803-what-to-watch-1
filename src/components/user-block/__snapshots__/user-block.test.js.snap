// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites component renders correctly 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="https://es31-server.appspot.com"
      width="63"
    />
  </div>
</div>
`;
